src/interfaces/business-state-interfaces.ts

```diff
@@ -28,7 +28,7 @@ export interface BusinessWarningIF {
   code: string // FUTURE: use an enum
   filing?: string // not used
   message: string
-  warningType: string // FUTURE: use an enum
+  warningType: WarningTypes
```

Comment:
As the number of warning types is continuing to grow, I thought this might be the time to finally do this 😁